app/models/vita_partner.rb

```diff
@@ -45,15 +45,21 @@ class VitaPartner < ApplicationRecord
   has_many :child_sites, -> { order(:id) }, class_name: "VitaPartner", foreign_key: "parent_organization_id"
 
   default_scope { includes(:child_sites).order(name: :asc) }
-
+  scope :with_capacity, lambda {
+    capacities = OrganizationCapacity.arel_table
+    joins(:organization_capacity).where(
+      capacities[:active_client_count].lteq(arel_table[:capacity_limit])
+          .or(arel_table[:capacity_limit].eq(nil)) # if capacity_limit is not set, assume org capacity
+    ).where.not(capacity_limit: 0)
+  }
   accepts_nested_attributes_for :source_parameters, allow_destroy: true, reject_if: lambda { |attributes| attributes['code'].blank? }
 
   def at_capacity?
```

Comment:
We are in favor of just using either the with_capacity scope or the at_capacity? method and getting rid of the other. Or rewriting `at_capacity?` to be a thin wrapper over the scope -- no more `if`s in `at_capacity?`. I personally think the at_capacity method is easier to read but understand that the scope might have computational benefits